ci/requirements-dev-3.6-main.yml

```diff
@@ -35,7 +35,7 @@ dependencies:
   - pymysql
   - pyspark>=2.4.3
   - pytables>=3.0.0
-  - pytest>=4.5
+  - pytest>=4.5, <=5.4.3
```

Comment:
you can do <6.0.0 i think

Reply:
`<6.0.0` doesn't seem to stop it from using 6.0.0rc1, but `<6.0.0rc1` works. I will change it to that